install/envs/mac.yml

```diff
@@ -33,6 +33,9 @@ dependencies:
   - psutil
   - kivy=2.0.0
   - plotly
+  - requests
+  - requests_toolbelt
```

Comment:
It seems we can't find this package in conda main and conda forge channels, maybe it's only available in pip?